src/EventListener/AddFormatListener.php

```diff
@@ -145,4 +151,17 @@ private function getNotAcceptableHttpException(string $accept, array $mimeTypes
             implode('", "', $mimeTypes)
         ));
     }
+
+    private function getRequestFormats(Request $request): array
+    {
+        return $this->isDocumentation($request)
+            ? $this->documentationFormats
+            : $this->formats
+        ;
+    }
+
+    private function isDocumentation(Request $request): bool
```

Comment:
Can you remove this function? (It's in the hot path, and function calls are costly).